x-pack/plugins/observability_log_explorer/README.md

```diff
@@ -1,3 +1,68 @@
 # Observability Log Explorer
 
 This plugin provides an app based on the `LogExplorer` component from the `log_explorer` plugin, but adds observability-specific affordances.
+
+## Testing
+
+### Stateful
+
+#### FTR Server
+```
+node scripts/functional_tests_server --config ./x-pack/test/functional/apps/observability_log_explorer/config.ts
```

Comment:
**nit:** I would suggest using the npm script `yarn test:ftr:server` and `yarn test:ftr:runner` for these references.
It doesn't make much more difference, but if in future the location of these scripts will change, the npm scripts will most likely remain the same, maintaining this reference still valid.